src/party_menu.c

```diff
@@ -5503,11 +5503,388 @@ void ItemUseCB_EvolutionStone(u8 taskId, TaskFunc task)
+                {
+                    task->fusionType = UNFUSE_MON;
+                    task->firstFusion = species;
+                    task->firstFusionSlot = gPartyMenu.slotId;
+                    task->fusionResult = itemFusion[i].fusingIntoMon;
+                    task->unfuseSecondMon = itemFusion[i].targetSpecies2;
+                    task->moveToLearn = itemFusion[i].fusionMove;
+                    task->forgetMove = itemFusion[i].unfuseForgetMove;
+                    TryItemUseFusionChange(taskId, taskFunc);
+                    return;
+                }
+            }
+            break;
+        case FUSE_MON:
+            if(task->fusionType == FUSE_MON) // Cancel If Second Mon is Another First Fusion Mon
+                break;
+            for (i = 0; itemFusion[i].targetSpecies1 != FORM_CHANGE_END; i++) // Run through the Fusion table for each species and check if the item matches one of the entries
+            {
+                if(itemFusion[i].itemId == gSpecialVar_ItemId)
+                {
+                    task->fusionType = FUSE_MON;
+                    task->firstFusion = species;
+                    task->firstFusionSlot = gPartyMenu.slotId;
+                    task->func = Task_HandleChooseMonInput;       
+                    gPartyMenuUseExitCallback = FALSE;
+                    sPartyMenuInternal->exitCallback = NULL;
+                    PartyMenuRemoveWindow(&sPartyMenuInternal->windowId[0]);
+                    DisplayPartyMenuStdMessage(PARTY_MSG_CHOOSE_SECOND_FUSION);
+                    return;
+                }
+            }
+            break;   
+        case SECOND_FUSE_MON:
+            if(task->fusionType != FUSE_MON) // Cancel if Secondary Fusion Mon Chosen First
+                break;
+            if(gPokemonStoragePtr->fusions[FusionStorageIndex(species)].level != 0)
+                break;
+            for (i = 0; itemFusion[i].targetSpecies1 != FORM_CHANGE_END; i++) // run through fusion table and check if the fusion works
+            {
+                if((itemFusion[i].itemId == gSpecialVar_ItemId) && (itemFusion[i].targetSpecies2 == species) && (itemFusion[i].targetSpecies1 == task->firstFusion))
+                {
+                    task->fusionResult = itemFusion[i].fusingIntoMon;
+                    task->secondFusionSlot = gPartyMenu.slotId;
+                    task->moveToLearn = itemFusion[i].fusionMove;
+                    // Start Fusion
+                    TryItemUseFusionChange(taskId, taskFunc);
+                    return;
+                }
+            }
+            break;
+    }
+    // No Effect Exit
+    gPartyMenuUseExitCallback = FALSE;                      
+    DisplayPartyMenuMessage(gText_WontHaveEffect, TRUE);    
+    ScheduleBgCopyTilemapToVram(2);                         
+    task->func = taskFunc;
+    return;
```

Comment:
Can you please fix all the syntax inconsistencies here?

Empty spaces between each `if` or `switch` and the opening parentheses, and in the case of the `switch`, the curly bracket needs to go in the next line, not at the end of the same line.